### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by mqiao2007** [zxzStar/pBEAM#2]
(1) adjust learning rate: e.g., decrease to 0.01, 0.05.
(2) adjust batch size: e.g., reduce from 20 to 10.
(3) add epochs: we are currently using one epoch. More epochs can be used.
